Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -200,15 +200,15 @@ internal async ValueTask<List<Result>> RequestUpdateAsync(string search, Cancell
                 Title = Context.API.GetTranslation("plugin_pluginsmanager_update_all_title"),
                 SubTitle = Context.API.GetTranslation("plugin_pluginsmanager_update_all_subtitle"),
                 IcoPath = icoPath,
-                Action = e => 
+                Action = e =>
                 {
                     // TODO: logic here
                     return true;
                 },
                 ContextData = new UserPlugin()
             };
 
-            var results = new[] {updateAllResult}.Concat(
+            var results = new[] { updateAllResult }.Concat(
```

Comment:
Maybe someone has an idea how to make this prettier. It works but I guess setting the top result in that way is not the most elegant solution :)